packages/contracts-router/contracts/test/Router.t.sol

```diff
@@ -0,0 +1,69 @@
+// SPDX-License-Identifier: MIT OR Apache-2.0
+pragma solidity 0.7.6;
+
+import "forge-std/Test.sol";
+import {RouterHarness} from "./harnesses/RouterHarness.sol";
+
+contract RouterTest is Test {
+    RouterHarness router;
+
+    uint32 testDomain;
+    bytes32 testRouter;
+
+    function setUp() public {
+        // deploy and initialize router
+        router = new RouterHarness();
+        address xAppConnectionManager = vm.addr(1321);
+        router.exposed__XAppConnectionClient_initialize(xAppConnectionManager);
+        testDomain = 123213;
+        testRouter = "router address";
+        // Make sure that Router is not enrolled yet
+        assertEq(router.remotes(testDomain), bytes32(0));
```

Comment:
i liked how you put this inside of the test case below. i think cleaner to do them all inside of the test case instead of here.